CHANGELOG.md

```diff
@@ -0,0 +1,58 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+
+    PROGRAM
```

Comment:
Make human readable or remove.

Reply:
wasn't it for other dev to know which files has been updated? It makes sense to me.

Reply:
It should be short and human readable. What files have been changed can easily be derived from git/